plugins/plugin-java-debugger/che-plugin-java-debugger-server/src/main/java/org/eclipse/che/plugin/jdb/server/JavaDebugger.java

```diff
@@ -203,6 +204,35 @@ public void start(StartAction action) throws DebuggerException {
   }
 
   @Override
+  public void runToLocation(RunToLocationAction action) throws DebuggerException {
+    lock.lock();
+    BreakpointImpl breakpoint =
+        new BreakpointImpl(
+            new LocationImpl(action.getTarget(), action.getLineNumber()),
+            new BreakpointConfigurationImpl(null, -1),
+            false);
+
+    try {
```

Comment:
Use one outer try-finally block and one inner block. Breakpoint creation should be inside outer block.